src/com/connectsdk/service/capability/MediaPlayer.java

```diff
@@ -82,6 +84,10 @@
 
     public void playMedia(MediaInfo mediaInfo, boolean shouldLoop, LaunchListener listener);
 
+    public void playMedia(MediaInfo mediaInfo, boolean shouldLoop, long startPosition, LaunchListener listener);
+
+    public void playMedia(MediaInfo mediaInfo, boolean shouldLoop, long startPosition, Object customData, LaunchListener listener);
```

Comment:
Because others custom data informations maybe needed at the service level, that are not related to the mediaInfo object. Take the [load](https://developers.google.com/android/reference/com/google/android/gms/cast/RemoteMediaPlayer#load(com.google.android.gms.common.api.GoogleApiClient, com.google.android.gms.cast.MediaInfo, boolean, long, long[], org.json.JSONObject)) method from the chromecast framework for instance.